tests/psoc6/dut/pin.py

```diff
@@ -0,0 +1,43 @@
+import os
+from machine import Pin
+import machine
+
+# Allocate pin based on board
+machine = os.uname().machine
+if "CY8CPROTO-062-4343W" in machine:
+    pin1_name = "P13_7"
+    pin2_name = "P13_6"
+elif "CY8CPROTO-063-BLE" in machine:
+    pin1_name = "P6_3"
+    pin2_name = "P6_2"
+
+# Pin out and pin in must be connected
+# together in the board
+
+pin_out = Pin(pin1_name)
+pin_out.init(Pin.OUT)
+pin_in = Pin(pin2_name, Pin.IN)
+
+pin_out.value(1)
+print("pin out value 1: ", pin_in.value() == 1)
+
+pin_out.value(0)
+print("pin out value 0: ", pin_in.value() == 0)
+
+pin_out.value(True)
+print("pin out value True: ", pin_in.value() == True)
+
+pin_out.value(False)
+print("pin out value False: ", pin_in.value() == False)
+
+pin_out.high()
+print("pin out value high: ", pin_in.value() == 1)
+
```

Comment:
Is on() and high() supposed to have same output? If I remember one was opposite way or?

Reply:
According to this docs -> high = on = 1? low = off = 0?
https://ifx-micropython.readthedocs.io/en/latest/library/machine.Pin.html#machine.Pin.on

Is that right? Or there is other conflicting definitions somewhere? 
Still this on() off() are inappropriate to me ... are belonging to the signal.

Reply:
I remember we had enabled this in opposite configuration back then because mpy docs suggested so (probably?). But since you have made change in machine_pins already, then this should be right. You can ignore my comment here.